docs/Inflationcalculator/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
     <title>Inflation calculator</title>
     <link rel="stylesheet" href="../style.css" />
+    <script src="./script.js"></script>
 </head>
 <body onload="updateSliderValue('current_cost_slider', 'current_cost_value'); updateSliderValue('rate_of_inflation_slider', 'rate_of_inflation_value'); updateSliderValue('time_period_slider', 'time_period_value'); calculateResult();">
```

Comment:
do we really require these function to be called? Check by removing.
Standard is you can call one function. But not these many functions' calls.